### PR TITLE
Styling for Pneumatic theme

### PR DESCRIPTION
**Pull request by helene-drivdigital**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1"/>
   <link rel="profile" href="http://gmpg.org/xfn/11">
   <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>">
+  <link href='//fonts.googleapis.com/css?family=Open+Sans:400,300,600' rel='stylesheet' type='text/css'>
```
Comment (forsvunnet) [inline header.php:10]: TODO: Må lages som en feature

File: index.php [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-  </article>
-<?php endwhile; endif; ?>
+    <?php if (have_posts()) : while (have_posts()) : the_post(); ?>
+      <article id="post-<?php the_ID(); ?>" <?php post_class( 'article' ); ?> role="article">
+        <div class="article--inner cf w">
+          <header class="article--header">
+            <?php the_post_thumbnail( 'large' ); ?>
+            <h1 class="article--title"><?php the_title(); ?></h1>
+            <?php do_action( 'pneumatic-theme-time', 'updated article-time' ); ?>
+          </header>
```
Comment (forsvunnet) [inline index.php:14]: TODO: Replace tabs with spaces

File: scss/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+$background-color: #fefefe;
+$menu-color: #eee;
+$menu-current: #ddd;
+$price-color: #c00;
+$footer-color: #eee;
+$badge-color: #4e4e4e;
+$star-color: #c00;
+
+/* Misc */
+$paragraph-size: 14px;
```
Comment (forsvunnet) [inline scss/_variables.scss:29]: TODO: Use em or % instead of px

Comment (yratof) [inline scss/_variables.scss:29]: Or rem, for accessibility

File: scss/breakpoints/_mobile-only.scss [hunk 4]
```diff
@@ -29,31 +27,15 @@
   }
 
   h1, h2, h3, h4, h5, h6 {
-    @extend %padd;
+    //@extend %padd;
```
Comment (forsvunnet) [inline scss/breakpoints/_mobile-only.scss:30]: TODO: re-enable this

File: scss/menus/_menu-1.scss [hunk 5]
```diff
@@ -146,15 +186,11 @@
   }
   .top-nav,
   .bar-nav {
-    background: #CCC;
+    background: #eee;
```
Comment (forsvunnet) [inline scss/menus/_menu-1.scss:189]: TODO: Use a variable instead

**Pull request merged**